lib/whitehall.rb

```diff
@@ -166,11 +166,12 @@ def self.edition_classes
       Publication,
       Speech,
       StatisticalDataSet,
+      *(EditionableWorldwideOrganisation if Flipflop.editionable_worldwide_organisations?),
     ]
   end
 
   def self.edition_route_path_segments
-    %w[news speeches policies publications consultations priority detailed-guides case-studies statistical-data-sets fatalities collections supporting-pages calls-for-evidence]
+    %w[news speeches policies publications consultations priority detailed-guides case-studies statistical-data-sets fatalities collections supporting-pages calls-for-evidence editionable-worldwide-organisations]
```

Comment:
Does this also need to be conditional on the feature flag?

Reply:
Um, this is used to check whether edition links are valid (part of the `ADMIN_EDITION_PATH` regex), so there is no user facing impact.

Do you think it should be behind the flag? I'm not sure if anything could go wrong tbh 🤔

Reply:
Ah, yes I see where this is used now.  Doesn't look like anything could go wrong by having it always included here.